src/ucp/tag/eager_snd.c

```diff
@@ -205,35 +205,37 @@ static ucs_status_t ucp_tag_eager_contig_short(uct_pending_req_t *self)
         return status;
     }
 
```

Comment:
Maybe we should pass the status to ucp_request_complete: we want to complete requests that are erroneous.